sdk/network/arm-network/CHANGELOG.md

```diff
@@ -1,15 +1,33 @@
 # Release History
+    
+## 32.0.0 (2023-07-06)
+    
+**Features**
 
-## 31.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Added Type Alias AdminState
+  - Interface ActiveDefaultSecurityAdminRule has a new optional parameter resourceGuid
+  - Interface ActiveSecurityAdminRule has a new optional parameter resourceGuid
+  - Interface AdminRule has a new optional parameter resourceGuid
+  - Interface AdminRuleCollection has a new optional parameter resourceGuid
+  - Interface ApplicationGateway has a new optional parameter defaultPredefinedSslPolicy
+  - Interface ConfigurationGroup has a new optional parameter resourceGuid
+  - Interface ConnectivityConfiguration has a new optional parameter resourceGuid
+  - Interface DefaultAdminRule has a new optional parameter resourceGuid
+  - Interface EffectiveConnectivityConfiguration has a new optional parameter resourceGuid
+  - Interface EffectiveDefaultSecurityAdminRule has a new optional parameter resourceGuid
+  - Interface EffectiveSecurityAdminRule has a new optional parameter resourceGuid
+  - Interface NetworkGroup has a new optional parameter resourceGuid
+  - Interface NetworkManager has a new optional parameter resourceGuid
+  - Interface SecurityAdminConfiguration has a new optional parameter resourceGuid
+  - Interface VirtualNetworkGateway has a new optional parameter adminState
+  - Added Enum KnownAdminState
+  - Class NetworkManagementClient has a new signature
 
-### Other Changes
+**Breaking Changes**
 
+  - Enum KnownApplicationGatewayCustomErrorStatusCode no longer has value HttpStatus499
```

Comment:
@kazrael2119 Could you help refer the breaking pr?

Reply:
[Release network microsoft.network official 2023 02 01 by htippanaboya · Pull Request #24277 · Azure/azure-rest-api-specs (github.com)](https://github.com/Azure/azure-rest-api-specs/pull/24277/commits/e5662c5f91b78f73f2e9b5e05bcd3c32a6b6148c)

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/24095 Remove 499 status code from custom error page